packages/crayons-core/src/index.html

```diff
@@ -10,9 +10,364 @@
 
     <script type="module" src="/build/crayons.esm.js"></script>
     <script nomodule src="/build/crayons.js"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600&display=swap"
+      rel="stylesheet"
+    />
   </head>
+  <style>
+    html .freshsales {
+      --fw-font-family: 'Montserrat';
+    }
+    body {
+      font-family: 'Montserrat', sans-serif;
+      margin: 0px;
+      background: #fff;
+    }
+    .container {
+      display: flex;
+      flex-wrap: wrap;
+    }
+    .container-row {
+      display: flex;
+      flex-direction: row;
+      flex-wrap: wrap;
+      width: 100%;
+      margin-bottom: 40px;
+    }
+    .container-col {
+      width: 40%;
+      padding: 10px 20px;
+      box-sizing: border-box;
+    }
+    .container-title {
+      width: 20%;
+      font-weight: 600;
+      padding: 10px 10px;
+      box-sizing: border-box;
+      display: flex;
+      align-items: center;
+    }
+    .container-header {
+      font-weight: 600;
+      display: flex;
+      align-items: center;
+    }
+    .freshsales fw-button {
+      --fw-button-height: 34px;
+      --fw-button-font-size: 12px;
+      --fw-button-font-weight: 400;
+      --fw-button-normal-line-height: 20px;
+      --fw-button-label-vertical-padding: 10px;
+      --fw-button-primary-color: #fff;
+      --fw-button-primary-border: #12344d;
+      --fw-button-primary-border-active: #264966;
+      --fw-button-primary-bg: #264966;
+      --fw-button-primary-bg-dark: #12344D;
+      --fw-button-primary-disabled-color: #fff;
+      --fw-button-primary-disabled-border: #83A6C8;
+      --fw-button-primary-disabled-bg: #A2C0DE;
+      --fw-button-primary-disabled-bg-dark: #83A6C8;
+      --fw-icon-size: 14px;
+      --fw-button-secondary-color: #333;
+      --fw-button-secondary-border: #CFD7DF;
+      --fw-button-secondary-border-active: #CFD7DF;
+      --fw-button-secondary-bg: #FFFFFF;
+      --fw-button-secondary-bg-dark: #F5F7F9;
+      --fw-button-secondary-disabled-color: #999999;
+      --fw-button-secondary-disabled-border: #CFD7DF;
```

Comment:
The value of this css variable is same as that of the default value. So can this be removed?